src/components/adaptive-cards/InputTime/InputTime.jsx

```diff
@@ -1,30 +1,94 @@
-import React from 'react';
+import React, {useContext, useEffect} from 'react';
 import PropTypes from 'prop-types';
-import {registerComponent} from '../Component/Component';
+import webexComponentClasses from '../../helpers';
+import {acPropTypes, registerComponent} from '../Component/Component';
+import {Select} from '../../generic';
+import AdaptiveCardContext from '../context/adaptive-card-context';
 
 /**
  * Adaptive Cards Input.Time component
  * https://adaptivecards.io/explorer/Input.Time.html
  *
  * @param {object} props  React props passed to the component
  * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Inline style for the component
  * @returns {object} JSX of the component
  */
-export default function InputTime({data}) {
-  const timeParts = data.value ? data.value.split(':') : [];
-  const hh = timeParts[0] || '';
-  const mm = timeParts[1] || '';
+export default function InputTime({data, className, style}) {
+  const [cssClasses] = webexComponentClasses('adaptive-cards-input-toggle', className);
+  const {setValue, getValue, setInput} = useContext(AdaptiveCardContext);
+  const timeParts = getValue(data.id).split(':');
+  const hours = (timeParts[0] || '').padStart(2, '0');
+  const minutes = (timeParts[1] || '').padStart(2, '0');
+  const hoursOptions = [...Array(24).keys()].map((a) => ({value: a, label: a}));
```

Comment:
the options also have to be zero-padded (instead of having 0, 1, 2, 3, to have '00', '01', '02', '03')

you can do this by defining a range function:

```js
function range(start, end) {
  return Array(end-start).keys().map((val) => start + val);
}

function pad2Zeros(value) {
  return String(value).padStart(2, '0');
}

const hoursOptions = range(0, 24).map(pad2Zeros).map((h) => {value: h, label: h});
```

You can put these 2 utility functions in util.js